### ISSUE TITLE
Increase size of buttons in Quote module

### ISSUE THREAD

**Issue by Nico-Kestra** [kestra-io/docs#4198]
### Describe the issue

In the Quote module, the “Read story”, “Previous”, and “Next” buttons are currently too small.

They should be increased in size to improve readability, usability, and accessibility. The current buttons are difficult to notice and interact with, especially on smaller screens.

<img width="2896" height="1192" alt="Image" src="https://github.com/user-attachments/assets/fe861f2f-d54a-4549-b9e5-aafe70af778d" />
